WooCommerce/WooCommerceTests/Tools/IntegerInputFormatterTests.swift

```diff
@@ -7,69 +7,69 @@ final class IntegerInputFormatterTests: XCTestCase {
 
     // MARK: test cases for `isValid(input:)`
 
-    func testEmptyInputIsValid() {
+    func test_isValid_with_empty_input_then_returns_true() {
```

Comment:
Thanks for updating the methods name using the new format! 

nit: I just say to not explicit the return type `then_returns_true`. It's not useful to know the return type. Some example below

Reply:
Indeed, refactored in https://github.com/woocommerce/woocommerce-ios/pull/6480/commits/cae78448b59a54bf647ffabe060be3b15eefa86f